pkg/cmd/pulumi/new.go

```diff
@@ -128,7 +128,9 @@ func runNew(ctx context.Context, args newArgs) error {
 
 	// Ensure the project doesn't already exist.
 	if args.name != "" {
-		if err := validateProjectName(ctx, b, args.name, args.generateOnly, opts); err != nil {
+		// No org name since
+		noOrgName := ""
+		if err := validateProjectName(ctx, b, args.name, noOrgName, args.generateOnly, opts); err != nil {
```

Comment:
optionally, you can use the pattern of inline `/* .. */` comments for documenting parameters here.

```suggestion
		noOrgName := ""
		if err := validateProjectName(ctx, b, args.name, "" /* org name */, args.generateOnly, opts); err != nil {
```

(e.g. https://github.com/uber-go/guide/blob/master/style.md#avoid-naked-parameters)

Reply:
Yeah. that's technically better because it decreases the number of identifiers added in the scope.